ranger/data/scope.sh

```diff
@@ -87,7 +87,15 @@ handle_extension() {
             ## Preview as text conversion
             odt2txt "${FILE_PATH}" && exit 5
             ## Preview as markdown conversion
-            pandoc -s -t markdown -- "${FILE_PATH}" && exit 5
+            ulimit -v 500000  # Limit to 500MB
```

Comment:
This doesn't work on macOS, `ulimit` can only set the maximum file size there.
Either way it would have to be reset afterwards though.
Other commands should be free to use however much memory they want if they can do so quickly and it should be possible for other alternatives to be tried after Pandoc.